sdv/metadata/metadata.py

```diff
@@ -324,6 +324,33 @@ def add_column(self, column_name, table_name=None, **kwargs):
         table_name = self._handle_table_name(table_name)
         super().add_column(table_name, column_name, **kwargs)
 
+    def remove_table(self, table_name):
+        """Remove a table from the metadata.
+
+        This method removes a table from the metadata as well as all relationships that table
+        contains.
+
+        Args:
+            table_name (str):
+                The name of the table to remove.
+        """
+        if table_name not in self.tables:
```

Comment:
Use this function:
https://github.com/sdv-dev/SDV/blob/efdaf855ad829111814fac221db0a028100ab3f9/sdv/metadata/multi_table.py#L400

Reply:
Updated. I will say that I think it's weird that this raises an `InvalidMetadataError` because the metadata isn't invalid. The user provided a bad input to a method on the metadata so it seems like it should be a `ValueError`